src/enums/bot-prefixes.enum.ts

```diff
@@ -1,6 +1,6 @@
 export enum BotPrefixesEnum {
-  LONG = "!hellforge",
-  AVERAGE = "!hell",
-  SHORT = "!hf",
-  TINY = "!h"
+  AVERAGE = `!hell`,
```

Comment:
are you talking about dick sizes?

Reply:
I- I don't see the point of that comment. Please refrain from making a dick joke publicly... :stuck_out_tongue: